pipelines/extension-build.yml

```diff
@@ -0,0 +1,43 @@
+pool:
+  vmImage: 'ubuntu-latest'
+
+trigger:
+  branches:
+    include:
+    - master
+  paths:
+    include:
+    - /extensions/
+    - /pipelines/
```

Comment:
Can we specify this specific pipeline instead of triggering on changes to any pipeline?